inc/saitypes.h

```diff
@@ -1221,6 +1233,9 @@ typedef union _sai_attribute_value_t
     /** @validonly meta->attrvaluetype == SAI_ATTR_VALUE_TYPE_INT32_RANGE */
     sai_s32_range_t s32range;
 
+    /** @validonly meta->attrvaluetype == SAI_ATTR_VALUE_TYPE_UINT16_RANGE_LIST */
+    sai_u16_range_list_t u16range_list;
```

Comment:
dont use underscore  to follow convention like in other items ? use "u16rangelist"

Reply:
Done